src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/SpecifyMarshalingForPInvokeStringArgumentsTests.Fixer.cs

```diff
@@ -139,6 +143,10 @@ End Sub
     <DllImport(""user32.dll"", CharSet:=CharSet.Unicode)>
     Private Shared Sub SomeMethod2(s As String)
     End Sub
+
+    <{|CA2101:DllImport(""user32.dll"", CharSet:=CharSet.Unicode)|}>
+    Private Shared Sub SomeMethod3(s As String)
+    End Sub
```

Comment:
I don't see why there is an expected CA2101 warning here? The above `SomeMethod2` looks exactly the same but don't expect violation.

Reply:
@Youssef1313 oops, that is a copy-paste error. Good catch. There shouldn't be any markup at all there as that's in the fixed-code. I've gone ahead and fixed it.